docs/8-the-supporting-acts/7-image-security.md

```diff
@@ -0,0 +1,140 @@
+
+1. Connect to the ACS WebUI:
+
+    ```bash
+    # get web url
+    echo https://$(oc -n rhacs-operator get route central --template='{{ .spec.host }}')
+    ```
+
+    Using the **admin** username:
+
+    ```bash
+    # get password to go with the "admin" username:
+    echo $(oc -n rhacs-operator get secret central-htpasswd -o go-template='{{index .data "password" | base64decode}}')
+    ```
+
+    ![images/acs-dashboard.png](images/acs-dashboard.png)
+    ![images/acs-compliance-graphs.png](images/acs-compliance-graphs.png)
+
+2. An API Token was created for us as part of the install configuration. We can retrieve it using:
+
+    Export the token as environment variable:
+
+    ```bash
+    export ROX_API_TOKEN=$(oc -n stackrox get secret rox-api-token-ml500 -o go-template='{{index .data "token" | base64decode}}')
+    ```
+
+    Export the StackRox endpoint:
+
+    ```bash
+    export ROX_ENDPOINT=central-stackrox.<CLUSTER_DOMAIN>
+    ```
+
+3. Verify the token by running **roxctl**
+
+    ```bash
+    roxctl central whoami --insecure-skip-tls-verify -e $ROX_ENDPOINT:443
+    ```
+
+4. This API token will be used by our pipeline. Let's create Sealed Secret definition for it.
+
+    ```bash
+    cat << EOF > /tmp/rox-auth.yaml
+    apiVersion: v1
+    data:
+      password: "$(echo -n ${ROX_API_TOKEN} | base64 -w0)"
+      username: "$(echo -n ${ROX_ENDPOINT} | base64 -w0)"
+    kind: Secret
+    metadata:
+      name: rox-auth
+    EOF
+    ```
+
+    Use `kubeseal` commandline to seal the secret definition.
+
+    ```bash
+    kubeseal < /tmp/rox-auth.yaml > /tmp/sealed-rox-auth.yaml \
+        -n <USER_NAME>-toolings \
+        --controller-namespace sealed-secrets \
+        --controller-name sealed-secrets \
+        -o yaml
+    ```
+
+    As always, we want to grab the results of this sealing activity, in particular the `encryptedData`. Because this is GitOps, and we will save it in our Git repos :)
+
+    ```bash
+    cat /tmp/sealed-rox-auth.yaml | grep -E 'username|password'
+    ```
+
+    <div class="highlight" style="background: #f7f7f7">
+    <pre><code class="language-yaml">
+        username: AgAj3JQj+EP23pnzu...
+        password: AgAtnYz8U0AqIIaqYrj...
+    </code></pre></div>
+
+    Open up `mlops-gitops/toolings/sealed-secrets/config.yaml` file and extend the Sealed Secrets entry. Copy the output of `username` and `password` from the previous command and update the values. Make sure you indent the data correctly.
+
+    ```yaml
+        - name: rox-auth
+            type: kubernetes.io/basic-auth
+            data:
+            username: AgAj3JQj+EP23pnzu...
+            password: AgAtnYz8U0AqIIaqYrj...
+    ```
+
+    Check our changes into git.
+
+    ```bash
+    cd /opt/app-root/src/mlops-gitops
+    git pull
+    git add .
+    git commit -m  "🔒 ADD - stackrox sealed secret 🔒"
+    git push
+    ```
+
+Now we can use ACS to help move security **LEFT** in our continous training pipeline. 
```

Comment:
Do you mean adding security earlier by "left"?

Reply:
yu the whole point of DevOps, shifting security left and all that jazz